common/txmgr/confirmer.go

```diff
@@ -802,17 +802,21 @@ func (ec *Confirmer[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) bum
 		return bumpedAttempt, err
 	}
 
-	if errors.Is(errors.Cause(err), commonfee.ErrBumpFeeExceedsLimit) {
+	if errors.Is(errors.Unwrap(err), commonfee.ErrBumpFeeExceedsLimit) {
```

Comment:
What are the differences between `Cause` and `Unwrap`?

Reply:
I would check with our tests to verify the behaviour, but Jordan is  right:
`Is reports whether any error in err's chain matches target. The chain consists of err itself followed by the sequence of errors obtained by repeatedly calling Unwrap.`